src/main/java/convobot/commands/Mark.java

```diff
@@ -0,0 +1,43 @@
+package commands;
+
+import exceptions.ConvoBotException;
+import utils.ResponseConstructor;
+import utils.TaskList;
+
+/**
+ * Represents a command to mark a task as completed in the ConvoBot application.
+ */
+public class Mark implements Command {
+
+    private final int i;
```

Comment:
This is not a temporary or iterator variable. It could have a longer name rather than just i?

Reply:
Accepted, renamed to indexToMark.